content/syllabuses/data-sci.md

```diff
@@ -51,6 +51,7 @@ title: "Data Science"
 - NCIT 14: Database Development
 - {{% contentlink "projects/sql/" %}}
 - NCIT 15: Analytics, Surveys, and Reports / Input & Output
+- {{% contentlink "projects/data-science-specific/logistic-regression/classification-assessment" %}}
```

Comment:
The link to the classification assessment is included on the logistic regression markdown - having it here seems extra